tests/test_checkout_caching.py

```diff
@@ -66,15 +75,15 @@ def test_aws_user_checkout_doesnt_create_tag(self):
         pass
```

Comment:
`pass`?  Must have missed this before.  Probably should be taken out.

Reply:
removed all references in file